### PR TITLE
Add django-pkgconf

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge if everything looks okay.

File: recipes/django-pkgconf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+about:
+  home: https://github.com/byashimov/django-pkgconf
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Yet another application settings helper'
+  description: 
+    Yet another application settings helper
+  doc_url: ttps://github.com/byashimov/django-pkgconf
```
Comment (dougalsutherland) [inline recipes/django-pkgconf/meta.yaml]: This and the next one are somehow missing an `h`.

File: recipes/django-pkgconf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{% set version = "0.3.0" %}
+{% set sha256 = "822bd778238828920d6f3b458a185e8b85d1eac689dddb1e2e38ec44234359ef" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/byashimov/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (dougalsutherland) [inline recipes/django-pkgconf/meta.yaml]: Is there a reason why you're pulling from github instead of pypi?

**Comment by dougalsutherland** [comment]
Okay, merged now. Thanks @sannykr.

**Pull request merged**